ProcessLib/SmallDeformation/CreateSmallDeformationProcess.cpp

```diff
@@ -112,6 +112,16 @@ createSmallDeformationProcess(
             std::copy(deactivated_subdomains_optional->begin(),
                       deactivated_subdomains_optional->end(),
                       std::back_inserter(deactivated_subdomains));
+            WARN(
+                "There are subdoamins being deactivated."
+                "Please make sure that there is no boundary condition "
+                "at any node inside these sudomains, "
+                "and only the iterative linear solver is utilized.");
+            INFO("The indices of the deactivated sudomains are");
```

Comment:
typo: subdomains.

Reply:
corrected by amending the corresponding commit.

Reply:
There are still few of the 'su_domains'